packages/common/src/store/ui/publish-playlist-confirmation-modal/selectors.ts

```diff
@@ -0,0 +1,4 @@
+import { CommonState } from 'store/commonStore'
+
+export const getPlaylistId = (state: CommonState) =>
```

Comment:
ah so i think this is definitely fine to do, having a separate state+slice+reducer for this, but one potential option is to use the "data" object in our modals, so you can maybe do `setVisible({name: "PublishPlaylist", data: {playlistId }})` then you can just leverage the existing modal strat and not have to have all the extra code around it. what you think?

Reply:
also this is not something we have established well yet, but i think another option could be to render the "drawer" right next to the component that needs it, and we can portal the drawer to the root. this would be a larger change overal, but a headsup as far as where we might want to go with this long-term